src/templates/containers/CommunityTemplatesIndex.tsx

```diff
@@ -64,6 +64,7 @@ import {event} from 'src/cloud/utils/reporting'
 const communityTemplatesUrl =
   'https://github.com/influxdata/community-templates#templates'
 const templatesPath = '/orgs/:orgID/settings/templates'
+const KEYBOARD_ENTER_KEY = 'Enter'
```

Comment:
nit: i think

```ts
if (event.key === 'Enter') {
  this.startTemplateInstall()
}
```

is pretty self documenting and this variable creates an abstraction that may not be necessary. perhaps not worrying about, but good to think about why we create abstractions...

Reply:
Ok, yeah. Let's go with the string literal.